android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapFragment.java

```diff
@@ -267,6 +268,7 @@ private void initCoverageTiles(String coverageUrl) {
                 mRootView.getContext(), coverageUrl, mMap);
         mCoverageTilesOverlayHighZoom = new CoverageOverlay(CoverageOverlay.LowResType.HIGHER_ZOOM,
                 mRootView.getContext(), coverageUrl, mMap);
+        mCoverageTilesOverlayDefaultZoom = new CoverageOverlay(mRootView.getContext(), coverageUrl, mMap);
```

Comment:
Instead of another constructor, CoverageOverlay.TileResType.UNMODIFIED_ZOOM or ORIGINAL_ZOOM